test/extensions/filters/http/common/fuzz/uber_filter.cc

```diff
@@ -94,6 +111,32 @@ void UberFilterFuzzer::fuzz(
   reset();
 }
 
+void UberFilterFuzzer::guideAnyProtoType(test::fuzz::HttpData* mutable_data, int seed) {
```

Comment:
small nit: can you pass in the actual index instead of the seed value? Just in case other filters extend this any to use a different set of services, mod 2 might grow to mod 3 and choosing the index by dividing by 2 would need to change.

Reply:
Done. Named it `choice` instead of index because it's not really the index in the array

Reply:
I think it's now divided by 4 by accident?

Reply:
Oops, good catch.